module/VuFind/src/VuFind/ILS/Logic/AvailabilityStatus.php

```diff
@@ -49,6 +49,13 @@ class AvailabilityStatus implements AvailabilityStatusInterface
      */
     protected int $availability;
 
+    /**
+     * Item visibility in holdings tab
+     *
+     * @var bool
+     */
+    protected bool $visibility;
```

Comment:
If you set this to true here, then you don't need an extra line in the constructor.